.github/workflows/gradle-run-tests.yml

```diff
@@ -20,13 +20,17 @@ jobs:
           cache: gradle
       - name: Validate Gradle wrapper
         uses: gradle/wrapper-validation-action@e6e38bacfdf1a337459f332974bb2327a31aaf4b
+      - name: Build Native dependencies
+        run: make
       - name: Test with Gradle
-        run: ./gradlew test
+        run: ./gradlew check
```

Comment:
I think if we just want to unit test on this step, we could manage to shave some minutes 👀  if we just run the test tasks directly, ie: `./gradlew :logic:test --max-workers 2 --parallel && ./gradlew :persistence:test --max-workers 2 --parallel` and so on. When we run the check task, I think it will also run for example linting and things that we could run on another gh action.

https://github.com/wireapp/kalium/runs/5494226737?check_suite_focus=true#step:6:663

Reply:
I found out the hard way that `test` will only run Android Unit tests.

The idea here is to actually run tests on all targets.

For example: run DB, crypto, network and logic tests on iOS, JVM, JS and on Android.

----

If we want to reduce tests to avoid having simple tests running on all targets, we could start moving some of the tests cases from `commonTest` (runs on all targets) to `jvmTest` (only on JVM).

----

To remove linting and other possible checks, I guess I'll try `testAll` instead.